### ISSUE TITLE
output.git.repo doesn't glob file path

### ISSUE THREAD

**Issue by nickhilliard** [ytti/oxidized#181]
the following config snippet:

```
output:
  default: git
  git:
    user: Oxidized
    email: oxidized@example.com
    repo: "~/oxidized/production.git"
```

... creates a new git repo in the current directory as follows:

```
/home/oxidised/.config/oxidized% ls -ald ./~/oxidized/production.git
drwxr-xr-x  6 oxidised  wheel  512 Sep 12 13:53 ./~/oxidized/production.git/
```

The principal of least astonishment would suggest that the ~ token should expand to the current user's home directoy.

**Comment by ytti** [comment]
https://github.com/ytti/oxidized/blob/master/lib/oxidized/output/git.rb#L18

Probably needs here File.expand_path ...

**Comment by ytti** [comment]
Fixed by c0a0089
